pkg/workload/workload.go

```diff
@@ -45,10 +46,14 @@ type PodSetResources struct {
 }
 
 func NewInfo(w *kueue.Workload) *Info {
-	return &Info{
+	info := &Info{
 		Obj:           w,
 		TotalRequests: totalRequests(&w.Spec),
 	}
+	if w.Spec.Admission != nil {
+		info.ClusterQueue = string(w.Spec.Admission.ClusterQueue)
```

Comment:
No, we were only using this field during scheduling, where the information comes from the queues, rather than the Workload.

Now I'm populating the field when adding the workload to the cache, for consistency and convenience (and to avoid potential bugs in the future).